script.js

```diff
@@ -13,25 +13,9 @@ function assignHendlerRadio() {
 assignHendlerRadio();
 
 function showRectangle(event) {
+  rectangleAll.forEach(item => item.style.opacity = '0');
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/voronkova-ab/game-bag/pullRequest?prid=7696171)
```suggestion
  rectangleAll.forEach((item) => item.style.opacity = '0');
```